src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPL_YOPERATION = "supply";
+    public static final String BUY_OPERATION = "buy";
+    public static final String DATA_SEPARATOR = " ";
+    public static final String CSV_SEPARATOR = ",";
+    public static final String RESULT = "result";
+    public static final int COLLUMN_INDEX_0 = 0;
+    public static final int COLLUMN_INDEX_1 = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(report(readFile(fromFileName)), toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = bufferedReader.readLine();
+            while (line != null) {
+                stringBuilder.append(line).append(" ");
+                line = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read this file...", e);
```

Comment:
Why was it changed? Previous version was correct
```suggestion
            throw new RuntimeException("Can't read from file " + fromFileName, e);
```